src/test/unittest/configurator.py

```diff
@@ -259,6 +259,19 @@ def ctx_choices(cls):
                             'TEST1, TEST2 and TEST5',
                             default='')
 
+        tracers = parser.add_mutually_exclusive_group()
+        tracers.add_argument('--tracer', dest='tracer', help='run C binary '
+                             'with provided tracer command. Unlike if binary '
```

Comment:
`With this option stdout and stderr are not redirected, enabling interactive sessions.`

Simpler is better.